src/main/java/com/redhat/lightblue/healthcheck/HealthCheckResource.java

```diff
@@ -78,7 +81,17 @@ public Response health() {
             assertEquals("updated", found.getValue());
 
             DataDeleteRequest deleteRequest = new DataDeleteRequest(ENTITY);
-            deleteRequest.where(Query.withValue("_id", Query.eq, uuid));
+            deleteRequest.where(
+                Query.or(
+                    Query.withValue("_id", Query.eq, uuid),
+                    Query.withValue("creationDate", Query.lte,
+                            Date.from(LocalDateTime
+                                    .now()
+                                    .minusDays(1)
```

Comment:
I would make this way shorter.  I estimate in a 24 hour period there is an opportunity to create 172,800 records given how often we expect this to be used (from load balancers) and how many nodes we have.

Recommend 5 minutes.